aspnetcore/fundamentals/openapi/samples/9.x/BuildTime/movie.cs

```diff
@@ -0,0 +1,11 @@
+using System.ComponentModel.DataAnnotations;
+
+public class Movie
+{
+    public int Id { get; set; }
+    public string? Title { get; set; }
+    [DataType(DataType.Date)]
```

Comment:
I wonder if there's a different way to show this concept, as at first glance I just thought "why not just use `DateOnly`?"